shared/actions/fs/platform-specific.desktop.js

```diff
@@ -258,10 +261,47 @@ function installCachedDokan(): Promise<*> {
   })
 }
 
-export function installDokanSaga() {
+function installDokanSaga() {
   return Saga.call(installCachedDokan)
 }
 
-export function copyToDownloadDir(path: string): Promise<*> {
-  return new Promise((resolve, reject) => resolve())
+function openFinderPopup(action: FsGen.OpenFinderPopupPayload) {
+  const {targetRect} = action.payload
+  return Saga.put(
+    navigateAppend([
+      {
+        props: {
+          targetRect,
+          position: 'bottom right',
+          onHidden: () => Saga.put(navigateUp()),
+          onInstall: () => Saga.put(FsGen.createInstallFuse()),
+        },
+        selected: 'finderAction',
+      },
+    ])
+  )
+}
+
+function* metaSaga(): Saga.SagaGenerator<any, any> {
+  yield Saga.safeTakeEveryPure(FsGen.openFinderPopup, openFinderPopup)
```

Comment:
why is `openFinderPopup` in `metaSaga` but all the other platform-specific sagas are in `subSaga`?